guides/common/modules/proc_verifying-dns-resolution.adoc

```diff
@@ -43,7 +43,7 @@ rtt min/avg/max/mdev = 0.019/0.019/0.019/0.000 ms
 ----
 
 ifdef::satellite[]
-For more information, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/networking_guide/index#sec_Configuring_Host_Names_Using_hostnamectl[Configuring Host Names Using hostnamectl] in the _{RHEL} 7 Networking Guide_.
+For more information, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/assembly_changing-a-hostname_configuring-and-managing-networking#proc_changing-a-hostname-using-nmcli_assembly_changing-a-hostname[Changing a hostname using nmcli] in the _{EL} 8 Configuring and managing networking_.
```

Comment:
I think it'd is confusing that above we describe `hostnamectl` but then link to `nmcli` while there's also a `hostnamectl` chapter. I'd link to that instead.

I'd also use `{RHEL}` instead of `{EL}` since it's the official RHEL documentation:
```suggestion
For more information, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/assembly_changing-a-hostname_configuring-and-managing-networking#proc_changing-a-hostname-using-nmcli_assembly_changing-a-hostname[Changing a hostname using nmcli] in the _{RHEL} 8 Configuring and managing networking_.
```

Note: I never knew `nmcli` could change the hostname.